app/assets/stylesheets/profiles.scss

```diff
@@ -3,32 +3,112 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 // Fixing different size images
 
-.pictures {
+.search {
+  @media print {
+    display: none;
+  }
+}
+
+.search__controls {
   display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
 }
 
-$profile-size: 250px;
+.search__neighborhood {
+  margin: 0.4rem;
+}
+
+.search__neighborhood-label {
+  margin: 0;
+}
+
+.search__button-container {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+}
+
+.search__button {
+  display: inline-block;
+  text-align: center;
+  flex: 1;
```

Comment:
Have you had a discussion about using flexbox on the site? Browser support [is pretty good](https://caniuse.com/#feat=flexbox) but IE doesn't handle it super well, so you'll either need to do some testing on IE to make sure it's ok, or decide that IE is unsupported (a decision that should be preceded by looking at the browser makeup of your site's users).

Reply:
@iangreenleaf We don't have any data on which browsers our users are on.  I have been very hesitant to do analytics, because I am not confident that we and future groups can keep that data safe.  What do you think?

Reply:
For now, we decided to not let IE support slow us down.  We will try to figure out some VM solution so we can test on it, but we are ok breaking things for a little while.